### DISCUSSION TITLE
Flashing message/banner

### DISCUSSION THREAD

**Discussion by asarkar** [mmistakes/minimal-mistakes#2650] [Q&A] [answered]
I recently changed my domain and wondering if the theme provides an easy way to flash a message/banner, saying "This site has moved to `https://newsite.com`". It doesn't have to be actually flashing, just clearly visible on the home page.

**Comment by iBug** [comment]
There's no such thing built into the theme. You can always create your own if you like.

For an easier start, try JavaScript `alert`. Insert this into your `index.md` or `index.html` somewhere:

```html
<script>alert("Hello world!");</script>
```

**Comment by asarkar** [comment]
JS pop ups are ugly and so 90s, so no :) If there's nothing OOTB, I'll look for customizing the home page. I was hoping this might be a common ask.

**Comment by mmistakes** [comment, marked as answer]
This is not a common ask. Closest thing  to what you want would be a notice.
https://mmistakes.github.io/minimal-mistakes/docs/utility-classes/#notices

Couldn't you add a redirect that points to the new site instead? That would be more transparent to the user and wouldn't require them to do anything either.

**Comment by asarkar** [comment]
> Couldn't you add a redirect that points to the new site instead?

I did already, but I'll not be renewing the old domain, and once it expires, the redirect will stop working. `.notice--warning` looks good for now.

**Comment by iBug** [comment]
@asarkar That's how the internet works `:)` Once you start running stuff on a domain, you're bound to it. I'd recommend keeping it up for at least a year before disposing of it.

**Comment by asarkar** [comment]
Just a note that the `.notice--warning` didn't work in `_layouts/home.html`; I used `<p class="notice--warning">` which did the job.
https://blog.asarkar.com/
